src-cpp/rdkafkacpp.h

```diff
@@ -1222,15 +1222,30 @@ class Queue {
    */
   static Queue *create (Handle *handle);
 
-        /**
-         * @brief      Forwards messages from this queue to another queue.
-         *
-         * @param      queue  The queue to forward to.
-         *
-         * @return     RdKafka::ERR_NO_ERROR on success or an error code on
-         *             error.
-         */
-        virtual ErrorCode forward (Queue *queue) = 0;
+  /**
+   * @brief Forward/re-route queue \p src to \p dst.
```

Comment:
there are no src and dst arguments.
Suggest renaming argument to dst and saying something like: Forward/re-route queue to \p dst.